composer.json

```diff
@@ -41,7 +41,7 @@
         "sebastian/code-unit": "^1.0.6",
         "sebastian/comparator": "^4.0.8",
         "sebastian/diff": "^4.0.3",
-        "sebastian/environment": "^5.1.3",
+        "sebastian/environment": "^5.1.4",
```

Comment:
I'm assuming [this PR](https://github.com/sebastianbergmann/environment/pull/66) will be released as that version.